CHANGELOG.md

```diff
@@ -2,45 +2,69 @@
 
 ## v6.0.0-beta.6 - 2017-08-24
 
+### Breaking change
+
+* Make invalid argument handling to -File and -Command consitent and make exit codes consistent with Unix standards (#4573)
+
+
 ### Engine updates and fixes
 
 * Make resource loading to work with PowerShell SxS installation (#4139)
 * Add missing assemblies to TPA list to make Pwrshplughin.dll work (#4502)
 * Make sure running `powershell` starts instance of the current version of PowerShell. (#4481)
+* Make sure we only use Unicode output by default on Nano and IoT systems (#4074)
+* Enable `-WindowStyle` to work. (#4573)
```

Comment:
Should say: powershell -windowstyle, otherwise it's not obvious.  Also mention only on Windows